manifests/database/postgresql.pp

```diff
@@ -41,21 +39,24 @@
   }
 
   if $init_db {
+    # Temporary direct use of liquibase to initially migrate the candlepin database
+    # until support is added in cpdb - https://bugzilla.redhat.com/show_bug.cgi?id=1044574
```

Comment:
Possibly, we could file an issue to do that in a separate PR when we've got the time.